demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -395,6 +402,18 @@ def insert_module_code(script_code: str, import_to_name: Dict[str, str]) -> str:
             script_code = script_code.replace(module_import, module_code)
         return script_code
 
+    @staticmethod
+    def insert_pack_version(script_code: str, pack_version: str) -> str:
```

Comment:
what happens if it's run on code which already has this value?

Reply:
Good point, fixed